parm/config/config.efcs

```diff
@@ -5,19 +5,36 @@
 
 echo "BEGIN: config.efcs"
 
+# TODO: the _ENKF counterparts need to be defined in config.base
+export DO_AERO=${DO_AERO_ENKF:-"NO"}
+export DO_OCN=${DO_OCN_ENKF:-"NO"}
+export DO_ICE=${DO_ICE_ENKF:-"NO"}
+export DO_WAVE=${DO_WAVE_ENKF:-"NO"}
+
+# TODO: Possibly need OCNRES_ENKF, ICERES_ENKF, WAVRES_ENKF too
+if [[ ${DO_OCN} == "YES" ]]; then
+  case "$CASE_ENKF" in
+    "C48") export OCNRES=500;;
+    "C96") export OCNRES=100;;
+    "C192") export OCNRES=050;;
+    "C384") export OCNRES=025;;
+    "C768") export OCNRES=025;;
```

Comment:
I think in the future, ideally, we should be able to have any combination and it should not depend on the atmosphere resolution.   Just like `CASE` is templated to `@CASE@`, I would think we would want a `@OCNRES@` etc. that gets filled when one runs `setup_expt.py`.  I think we can save that for a future PR.
As soon as we start doing cycling with an ensemble, this will have to be updated as C96 sets a ocean resolution different that C48.

Reply:
That sounds great.  Unfortunately there are files that depend on CASE and OCN RES so you can't have "any" combination without the appropriate fix files, but ideally it wouldn't be pre-determined otherwise and I like the idea of the wave grid being an independent choice.

Reply:
Yes.  When we provide that flexibility, we will also implement a "validity check" to ensure that the the user has the illusion of choice, but not really :-)